pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/LuceneRealtimeClusterIntegrationTest.java

```diff
@@ -58,6 +58,12 @@
 
   private static final String TEST_TEXT_COLUMN_QUERY =
       "SELECT COUNT(*) FROM mytable WHERE TEXT_MATCH(skills, '\"machine learning\" AND spark')";
+  private static final String TEST_CONTAINS_QUERY =
```

Comment:
Don't add it into this integration test. We can add some tests to the `CalciteSqlCompilerTest` for the syntax support

Reply:
I would still argue that we introduce the Lucene index support in this PR. We should not disallow CONTAINS if the user does not have a native text index, but has a Lucene text index